apps/contracts/contracts/storage/ZengoStorage.sol

```diff
@@ -19,4 +19,13 @@ contract ZengoStorage {
     mapping(address => Structs.Moderator) moderatorStruct;
 
     mapping(address => uint256) public votingPoints;
+
+    function getModeratorInfo(address _moderator) public view returns (Structs.Moderator memory) {
```

Comment:
@Markkos89  this one

Reply:
> @Markkos89 this one

This one only returns the Mod Info **by address**

Reply:
Yes

Reply:
Added a getAllModerators function